tests/GameCards.spec.ts

```
@@ -95,5 +95,65 @@ describe('GameCards', function() {
     const names = new GameCards(gameOptions).getProjectCards().map((c) => c.name);
     expect(names).to.contain(CardName.VENUSIAN_INSECTS);
   });
+
+  it('should not include the included cards in the standard projects', function () {
+    const gameOptions: GameOptions = {
+      ...DEFAULT_GAME_OPTIONS,
+      corporateEra: true,
+      includedCards: [CardName.VENUSIAN_INSECTS],
+    };
+    const names = new GameCards(gameOptions).getStandardProjects().map((c) => c.name);
+    expect(names).to.not.contain(CardName.VENUSIAN_INSECTS);
+  });
+
+  it('should not include the included cards in the preludes', function () {
+    const gameOptions: GameOptions = {
+      ...DEFAULT_GAME_OPTIONS,
+      corporateEra: true,
+      includedCards: [CardName.VENUSIAN_INSECTS],
+    };
+    const names = new GameCards(gameOptions).getPreludeCards().map((c) => c.name);
+    expect(names).to.not.contain(CardName.VENUSIAN_INSECTS);
+  });
+
+  it('should not include the included cards in the corporation cards', function () {
+    const gameOptions: GameOptions = {
+      ...DEFAULT_GAME_OPTIONS,
+      corporateEra: true,
+      includedCards: [CardName.VENUSIAN_INSECTS],
+    };
+    const names = new GameCards(gameOptions).getCorporationCards().map((c) => c.name);
+    expect(names).to.not.contain(CardName.VENUSIAN_INSECTS);
+  });
+
+  it('should not include corporation cards in the included cards', function () {
```

I didn't let it throw an error as in your example, but just filter out all non project cards that people might have manually added in a settings json file.

---

OIK good